@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp

```diff
@@ -890,7 +890,37 @@ class CfgVehicles
                isRadioRemovable = 0;
                intercom[] = {};                   // All units in intercom will be able to hear/send transmittions (ACE3 interaction menu) but they cannot manipulate the radio (GUI interface)
            };
-       };
+        };
+		class Attributes
+		{
+			class fza_pylonLoadout
+			{
+				displayName = "Loadout Config"; // Name assigned to UI control class Title
+				tooltip = "Loadout in JSON form to export"; // Tooltip assigned to UI control class Title
+				property = "fza_ah64_loadout"; // Unique config property name saved in SQM
+				control = "Edit"; // UI control base class displayed in Edit Attributes window, points to Cfg3DEN >> Attributes
+
+				// Expression called when applying the attribute in Eden and at the scenario start
```

Comment:
Can we remove these comments as discussed please? It is relatively self-explanatory, is a duplication of the docs and clutters the actual content.